Pipfile

```diff
@@ -14,7 +14,7 @@ djangorestframework-csv = "*"
 bleach = "*"
 newrelic = "*"
 cfenv = "*"
-cg-django-uaa = "*"
+cg-django-uaa = {git = "https://github.com/18F/cg-django-uaa",ref = "v2.0.0"}
```

Comment:
What do you think about logging an issue to ourselves to update this when cg-django-uaa gets a proper release?